client/my-sites/checkout/checkout-thank-you/domains/thank-you-content/domain-registration.tsx

```diff
@@ -24,33 +25,64 @@ const DomainRegistrationThankYouProps = ( {
 			selectedSiteSlug,
 		},
 		'REGISTRATION',
-		false
+		true
 	);
 
 	const returnProps: DomainThankYouProps = {
+		thankYouNotice: {
+			noticeTitle: translate(
+				'It may take up to 30 minutes for your domain to start working properly.'
+			),
+			noticeIconCustom: <Icon icon={ info } size={ 24 } />,
+		},
 		sections: [
 			{
 				sectionKey: 'domain_registration_whats_next',
 				sectionTitle: translate( 'What’s next?' ),
 				nextSteps: [
+					...( professionalEmail ? [ professionalEmail ] : [] ),
+					...( ! selectedSiteSlug
+						? [
+								{
+									stepKey: 'domain_registration_whats_next_create-site',
+									stepTitle: translate( 'Add a site' ),
+									stepDescription: translate( 'Choose a theme, customize and launch your site.' ),
+									stepCta: (
+										<FullWidthButton
+											href={ createSiteFromDomainOnly( domain, null ) }
+											busy={ false }
+											disabled={ false }
+										>
+											{ translate( 'Create a site' ) }
+										</FullWidthButton>
+									),
+								},
+						  ]
+						: [] ),
 					{
-						stepKey: 'domain_registration_whats_next_plugin_setup',
-						stepTitle: translate( 'Organize your domains' ),
-						stepDescription: translate(
-							'Set up a primary domain, connect other domains and make sure people can find your site'
-						),
+						stepKey: 'domain_registration_whats_next_view_domains',
+						stepTitle: selectedSiteSlug
+							? translate( 'Organize your domains' )
+							: translate( 'Manage your domains' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **52** times:
`translate( 'Manage Your Domains' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/domain-only-thank-you-page)